### PR TITLE
cherry-pick command added

### PR DESCRIPTION
**Pull request by prakashio**
Cherry-pick command added which

### CODE REVIEW

File: src/data/primary-options.js [hunk 1]
```diff
@@ -17,7 +17,8 @@ const options = [
   { value: 'debug', label: 'debug' },
   { value: 'recover', label: 'recover' },
   { value: 'synchronize', label: 'synchronize' },
-  { value: 'rebase', label: 'rebase' }
+  { value: 'rebase', label: 'rebase' },
+  { value: 'cherrypick', label: 'cherry-pick' },
```
Comment (KoltonG) [inline src/data/primary-options.js]: What do you think of adding this under the following steps to make it easier for newer users? 👇
- Dropdown 1: "Add"
- Dropdown 2: "A commit from another branch"

File: src/data/secondary-options.js [hunk 2]
```diff
@@ -470,5 +470,15 @@ export const secondaryOptions = {
...
       usage: 'git stash branch <branch name> <stash id>'
     }
-  ]
+  ],
+
+  cherrypick: [
+    {
+      value: 'origin-branch',
+      label: 'an commit from origin branch into my working branch',
+      usage: 'git cherry-pick <commit-hash> <commit-hash>',
```
Comment (KoltonG) [inline src/data/secondary-options.js]: I was looking at this documentation for [Cherry Pick](https://www.previousnext.com.au/blog/intro-cherry-picking-git) and I think you only need one commit-hash?
```suggestion
      usage: 'git cherry-pick <commit-hash>,
```

File: src/data/secondary-options.js [hunk 3]
```diff
@@ -470,5 +470,15 @@ export const secondaryOptions = {
...
+      nb:
+        'Applying one or more commit from one branch into your working branch. '
```
Comment (KoltonG) [inline src/data/secondary-options.js]: The user will need to know how to get the **commit hash** from the source branch and make sure that they are on the **destination branch** before running this command.

File: src/data/primary-options.js (hunk 1 above)
Comment (Prakash106) [reply src/data/primary-options.js]: It would be better.

File: src/data/secondary-options.js (hunk 2 above)
Comment (Prakash106) [reply src/data/secondary-options.js]: Multiple commit is also possible if the user want to cherry-pick more than one commit

**Pull request merged**